account_bank_receipt/views/account_bank_receipt_view.xml

```diff
@@ -0,0 +1,136 @@
+<odoo>
+
+    <record id="account_bank_receipt_view_form" model="ir.ui.view">
+        <field name="model">account.bank.receipt</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button name="validate_deposit"
+                            states="draft"
+                            string="Validate"
+                            type="object" class="oe_highlight"/>
+                    <button name="backtodraft"
+                            states="done"
+                            string="Back to Draft"
+                            type="object"/>
+                    <button name="get_report" string="Print" type="object"/>
+                    <field name="state"
+                           widget="statusbar"
+                           statusbar_visible="draft,done"/>
+                </header>
+                <sheet>
+                    <div class="oe_title">
+                        <label string="Bank Receipt" for="name"/>
+                        <h1>
+                            <field name="name"/>
+                        </h1>
+                    </div>
+                    <group name="main">
+                        <group name="left">
+                            <field name="partner_id"/>
+                            <field name="receipt_date"/>
+                            <field name="journal_id"
+                                   widget="selection"/>
+                            <field name="journal_default_account_id"
+                                   invisible="1"/>
+                            <field name="currency_id"
+                                   groups="base.group_multi_currency"/>
+                            <field name="bank_journal_id" widget="selection"/>
+                        </group>
+                        <group name="right">
+                            <field name="company_id"
+                                   groups="base.group_multi_company"/>
+                            <field name="currency_none_same_company_id"
+                                   invisible="1"/>
+                            <field name="check_count" invisible="1"/>
+                            <field name="total_amount" widget="monetary"
```

Comment:
For v12, I think we better use fields.Monetary() and don't need to specify in view anymore